<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PinWidget</class>
 <widget class="QWidget" name="PinWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QListWidget" name="stepList"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QPushButton" name="addStep">
     <property name="text">
      <string>Pin current Step</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="showStep">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Display Step</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="showCell">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Display Cell</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="delStep">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Delete pinned Step</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QPushButton" name="insertStep">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Insert pinned into current Step</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
